### ISSUE TITLE
V1 : Error importing dxf

### ISSUE THREAD

**Issue by ibgcia** [jscad/OpenJSCAD.org#531]
Hi.

I'm trying to import a dxf file and I'm getting the next error message:

> Uncaught TypeError: Cannot read property 'class' of undefined
Line: 88,col: 23
(https://openjscad.org/dist/index.js)

I've been trying with several dxf and I got the same result. Even I've opened a native example on the web, I've converted it and when I was going to import it I got the same error. One of the dxf's is the next. It's an dxf exported by the openjscad web.

[output (3).txt](https://github.com/jscad/OpenJSCAD.org/files/4322536/output.3.txt)

Any suggestions?

**Comment by z3dev** [comment]
Are you working with 2D shapes at all?

**Comment by rgrastur** [comment]
> Are you working with 2D shapes at all?
Hi z3dev, What do you mean with working 2D shapes?.  

I have the same result when I download one dxf generated example (is a 3D model), for example a cone with cutouts, downloaded in examples section.
![image](https://user-images.githubusercontent.com/48772925/76733458-25affa80-6761-11ea-9be3-fb2dc9112338.png)


When I drop it in the box below. I get an error with a message at the top of view but never completes.

![image](https://user-images.githubusercontent.com/48772925/76733584-63ad1e80-6761-11ea-851a-581507453c59.png)

**Comment by z3dev** [comment]
Looks like you found a bug in the export for DXF. It's adding some 2D entities to the DXF file.

And when the DXF file is imported again, those 2D entities are being converted, but incorrectly.

```
  let jscad1 = new CSG.Path2D()
  jscad1 = jscad1.appendPoint( [-0.5,-0.5] ).appendPoint( [0.5,0.5] )
  let jscad2 = CSG.Line2D.fromPoints(new CSG.Vector2D(-1,0.26794919),new CSG.Vector2D(0,0))
  let jscad3 = CSG.Line2D.fromPoints(new CSG.Vector2D(0,0),new CSG.Vector2D(-1,-0.26794919))
  let jscad4 = CSG.Line2D.fromPoints(new CSG.Vector2D(0,0),new CSG.Vector2D(-1,0))
```

You can probably get around this by removing those unwanted 2D shapes.

**Comment by z3dev** [comment]
Those unwanted 2D entities are also present in V2 DXF files.

**Comment by z3dev** [comment]
Fixed in V2, pull request #537
